lib/puppet/type/foreman_smartproxy.rb

```diff
@@ -33,6 +33,14 @@
     newvalues(URI.regexp)
   end
 
+  newproperty(:organizations, :array_matching => :all) do
+    desc 'The organizations to add to the smartproxy'
```

Comment:
>  2) smart proxy (two words)

The rest of the parameters say smartproxy, this was for consistency https://github.com/stbenjam/puppet-foreman/blob/766cce57eade60da5e347f18b18497eb9bb0f092/lib/puppet/type/foreman_smartproxy.rb#L7